.github/workflows/main.yml

```diff
@@ -16,8 +16,10 @@ jobs:
         uses: ./
         with:
           config: fixtures/cliff.toml
-          args: --verbose --strip 'footer' --exclude-path '.github/**'
+          args: --verbose --strip 'footer' --exclude-path '.github/**' --tag 0.0.0
```

Comment:
Added this so that we see the version string in the test run, otherwise, it would appear empty.